x-pack/plugins/asset_inventory/tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+  "extends": "../../../tsconfig.base.json",
+  "compilerOptions": {
+    "outDir": "./target/types"
+  },
+  "include": [
+    "common/**/*.ts",
+    "common/**/*.json",
+    "public/**/*.ts",
+    "public/**/*.tsx",
+    "public/**/*.json",
+    "server/**/*.ts",
+    "server/**/*.json",
+    // "../../typings/**/*",
```

Comment:
To-do: Remove comment

Reply:
Just thinking if it would be easier to exclude. 🤔